.changelog/45464.txt

```diff
@@ -0,0 +1,9 @@
+```release-note:enhancement
+provider: The `provider_meta` block is now supported. The `user_agent` argument enables module authors to include additional product information in the `User-Agent` header sent during all AWS API requests made during Create, Read, Update, and Delete operations.
```

Comment:
Link to https://developer.hashicorp.com/terraform/internals/provider-meta as most won't be familiar with `provider_meta` block.